src/Units/Unit.java

```diff
@@ -253,8 +253,9 @@ private static int healthFromHP(double input)
    */
   public int damageHP(double damage)
   {
+    if( damage < 0 ) throw new ArithmeticException("Cannot inflict negative damage!");
```

Comment:
Hah. Guess I'm glad we threw out the idea of Brenner healing 1.5 HP...

Reply:
Hm? `AlterHP` can still be used to add HP. Or was he going to do that by attacking? That would be weird.

Reply:
Right. It didn't really fit that well, which is why it was scrapped.